pkg/driver/grpc.go

```diff
@@ -46,13 +47,29 @@ func parseEndpoint(ep string) (string, string, error) {
 // logGRPC logs all the grpc related errors, i.e the final errors
 // which are returned to the grpc clients
 func logGRPC(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
-	logrus.Infof("GRPC call: %s", info.FullMethod)
-	logrus.Infof("GRPC request: %s", protosanitizer.StripSecrets(req))
+
+	// add the messages that pollute logs to the array
+	var msgsToFilter = [][]byte{
+		[]byte("NodeGetVolumeStats"),
+		[]byte("NodeGetCapabilities"),
+	}
+
+	// checks for message in request
+	for _, msg := range msgsToFilter {
+		if bytes.Contains([]byte(info.FullMethod), msg) {
+			resp, err := handler(ctx, req)
+			return resp, err
```

Comment:
we should always call the handler at line 68. Here the intention is to avoid the logs, returning from this place is not right.

create a filter function which should return true/false based on the which grpc should get filtered. And check that return value to print the logs. No matter what, the handler should always gets called.

Reply:
so that means that no matter we will always print the GRPC response ?  just the call & request will be removed ? or should that too depend on the returned boolean ?

Reply:
yep ,response should also depend on that.

Reply:
is it better now ?

Reply:
better. Thanks.